src/platforms/windows/daemon/windowssplittunnel.cpp

```diff
@@ -484,6 +484,9 @@ bool WindowsSplitTunnel::getAddress(int adapterIndex, IN_ADDR* out_ipv4,
     logger.debug() << "Ipv4 Conversation error" << WSAGetLastError();
     return false;
   }
+  if (ipv6.empty()) {
```

Comment:
As far as I can tell, there is nothing that explicitly initializes the contents of `out_ipv6` when we follow this code path. It probably works in a debug build because it defaults to all zeros, but that may not be true when compiled for release. I would recommend setting some kind of value here to ensure we are not passing uninitialized data to the caller.

Reply:
Good catch! Did that.